tensorflow/python/ops/math_ops.py

```diff
@@ -386,6 +386,26 @@ def erf(x, name=None):
     else:
       return gen_math_ops.erf(x, name=name)
 
+    
+@tf_export("math.erfinv", "erfinv")  
```

Comment:
@annarev do we want to add it in the root namespace for consistency with erf, or should we just add it in math?

Reply:
Sorry, I missed the question earlier. We are going to deprecate tf.erf and only keep tf.math.erf in TensorFlow 2.0. So, it is best to export erfinv as `@tf_export("math.erfinv")`